app/data/dbConn.py

```diff
@@ -1,22 +1,24 @@
 import sqlite3
 from sqlite3 import Error
+from app.logs.logger import logger
 
-from app import app_config
 
+from app import app_config
+from app import app
 
 def connect():
 
     try:
         # SQLite objects created in a thread can only be used in that same thread.
         # tech_same_thread=False repaired it
         connection = sqlite3.connect(app_config.DATABASE_PATH, check_same_thread=False)
-        print(sqlite3.version)
 
+        logger.debug("Successfully connected to database")
         return connection
 
     except Error as e:
-        print(e)
-
+        logger.critical("Connection to database failed", exc_info=True)        
```

Comment:
stop app when that happens?

Reply:
I believe I requested that already, try not to commit whitespace changes that would outdate comment

Reply:
yes, but wanted to do this later, will try to implement it

Reply:
Tried but didn't manage to do this

Reply:
add todo at least, it has to be done eventually